google-cloud-debugger/ext/google/cloud/debugger/debugger_c/evaluator.h

```diff
@@ -19,6 +19,14 @@
 
 #include "ruby/debug.h"
 
+// Cache commonly used Ruby IDs
+#define LOCALS_ID rb_intern("locals")
```

Comment:
This just does a textual replacement. It still makes the call to rb_intern each time, so it's not really helping. Consider instead creating a static global for each of these and just referencing it. Or, probably better, use CONST_ID or similar macro to memoize.